From d924e23fa60023bdbc601797ccda30c2c341a34a Mon Sep 17 00:00:00 2001
From: Bai Ping <b51503@freescale.com>
Date: Mon, 14 Sep 2015 18:30:43 +0800
Subject: [PATCH] MLK-11537 ARM: dts: add setpoint setting for
 imx6ul-14x14-lpddr2-arm board

On i.MX6UL, the VDD_ARM_IN and VDD_SOC_IN use the same power rail
VDD_ARM_SOC_IN. For imx6ul-14x14-lpddr2-arm2 board, default, we use
ldo-bypass mode and use the external PMIC regulator for cpufreq.so
on this board, should use the setpoint of ldo-bypass mode.

Signed-off-by: Bai Ping <b51503@freescale.com>
Signed-off-by: Peng Fan <Peng.Fan@freescale.com>
---
 arch/arm/boot/dts/imx6ul-14x14-lpddr2-arm2.dts | 18 ++++++++++++++++++
 1 file changed, 18 insertions(+)

diff --git a/arch/arm/boot/dts/imx6ul-14x14-lpddr2-arm2.dts b/arch/arm/boot/dts/imx6ul-14x14-lpddr2-arm2.dts
index 281c25b..3bee66a 100644
--- a/arch/arm/boot/dts/imx6ul-14x14-lpddr2-arm2.dts
+++ b/arch/arm/boot/dts/imx6ul-14x14-lpddr2-arm2.dts
@@ -98,6 +98,24 @@
 };
 
 &cpu0 {
+	/*
+	 * on i.MX6UL, no seperated VDD_ARM_IN and VDD_SOC_IN,
+	 * to align with other platform and use the same cpufreq
+	 * driver, still use the seperated OPP define for arm
+	 * and soc.
+	 */
+	operating-points = <
+		/* kHz	uV */
+		528000	1175000
+		396000	1175000
+		198000	1175000
+	>;
+	fsl,soc-operating-points = <
+		/* KHz	uV */
+		528000	1175000
+		396000	1175000
+		198000	1175000
+	>;
 	arm-supply = <&sw1a_reg>;
 	soc-supply = <&sw1a_reg>;
 	fsl,arm-soc-shared = <1>;
-- 
1.8.0

